.github/workflows/vendor_gardener.yaml

```diff
@@ -0,0 +1,31 @@
+name: Vendor Gardener
```

Comment:
In https://github.com/gardener/gardener-extension-registry-cache/pull/147/commits/f683d23a862bd1473ebc30ddf118213247ccfb99, I dropped this Github action with the vendor dir removal. I think the only use of this Github action after the vendor dir removal would be to update the 2 script that are currently coming from gardener (component_descriptor and prepare_release). I decided to manually maintain these 2 scripts instead of having such Github action.

No strong opinion, I just wanted to share the approach I decided to follow for the registry-cache repo.

Reply:
Yep I noticed that, but since I sometimes tend to forget such stuff, I decided to leave the action :)